src/main/plugins/org.dita.pdf2/xsl/fo/loc.xsl

```diff
@@ -0,0 +1,231 @@
+  <xsl:variable name="changeset" select="//*[contains (@class, ' relmgmt-d/change-item ')]"/>
+
+  <!-- The variable topicsWithChangeItem contains a list of all topics containing change-item elements. -->
+  <xsl:variable name="topicsWithChangeItem" select="//*[contains (@class, ' topic/topic ')][.//*[contains (@class, ' relmgmt-d/change-item ')]]"/>
+
+  <!-- Keep date format in a variable -->
+  <xsl:variable name="dateFormat">[MNn] [D], [Y]</xsl:variable>
+
+  <!--   LOC   -->
+  <xsl:template match="ot-placeholder:changelist" name="createChangeList">
+    <xsl:if test="//*[contains(@class, ' relmgmt-d/change-historylist ')]">
+      <fo:page-sequence master-reference="toc-sequence" xsl:use-attribute-sets="page-sequence.loc">
+        <xsl:call-template name="insertTocStaticContents"/>
+        <fo:flow flow-name="xsl-region-body">
+          <fo:block start-indent="0in">
+            <xsl:call-template name="createLOCHeader"/>
+            <xsl:apply-templates select="//*[contains(@class, ' topic/critdates ')][1]"/>
+          </fo:block>
+        </fo:flow>
+      </fo:page-sequence>
+    </xsl:if>
+  </xsl:template>
+
+  <xsl:template name="createLOCHeader">
+    <fo:block xsl:use-attribute-sets="__lotf__heading" id="{$id.lot}">
+      <fo:marker marker-class-name="current-header">
+        <xsl:call-template name="getVariable">
+          <xsl:with-param name="id" select="'List of Changes'"/>
+        </xsl:call-template>
+      </fo:marker>
+      <xsl:call-template name="getVariable">
+        <xsl:with-param name="id" select="'List of Changes'"/>
+      </xsl:call-template>
+    </fo:block>
+  </xsl:template>
+  
+  <xsl:template match="//*[contains(@class, ' bookmap/bookmeta ')]/*[contains(@class, ' topic/critdates ')]">
+
+    <!-- Create a list of all book releases -->
+    <xsl:variable name="bookReleaseDates" select="*[contains (@class, ' topic/created ')]/@date |
+                                                  *[contains (@class, ' topic/revised ')]/@modified"/>
+    <xsl:variable name="bookReleaseString">
+      <xsl:call-template name="getVariable">
+        <xsl:with-param name="id" select="'Book Release'"/>
+      </xsl:call-template>
+    </xsl:variable>
+    <xsl:variable name="bookReleaseDateString">
+      <xsl:call-template name="getVariable">
+        <xsl:with-param name="id" select="'Book Release Date'"/>
+      </xsl:call-template>
+    </xsl:variable>
+    <xsl:variable name="colonSymbol">
+      <xsl:call-template name="getVariable">
+        <xsl:with-param name="id" select="'ColonSymbol'"/>
+      </xsl:call-template>
+    </xsl:variable>
+    <xsl:variable name="separator">
+      <xsl:call-template name="getVariable">
+        <xsl:with-param name="id" select="'figure-number-separator'"/>
+      </xsl:call-template>
+    </xsl:variable>
+
+    <!--<xsl:for-each select="1 to (count($bookReleaseDates) - 1)">-->
+    <xsl:for-each select="$bookReleaseDates ">

```

Comment:
`$bookReleaseDates` is two attribute nodes. Replace index based for-loop with current node based approach.